en/sensor/px4flow.md

```diff
@@ -1,467 +1,10 @@
-# PX4FLOW Smart Camera
+# PX4FLOW Smart Camera (Deprecated)
```

Comment:
Page renders like this. @mcsauder I assume that we're indicating that PX4 v1.13 is the last version it might work on with old PX4Flow firmware (not the current lot)?


![image](https://user-images.githubusercontent.com/5368500/232970723-d0b23474-9172-411b-8a0f-e59dd9e88a0f.png)